src/couch_replicator_scheduler.erl

```diff
@@ -266,57 +260,24 @@ remove_job_int(#job{} = Job) ->
     ets:delete(?MODULE, Job#job.id).
 
 
--spec job_count() -> non_neg_integer().
-job_count() ->
-    ets:info(?MODULE, size).
-
-
 -spec running_job_count() -> non_neg_integer().
 running_job_count() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        [true]}
-    ],
-    ets:select_count(?MODULE, MatchSpec).
-
+    ets:info(?MODULE, size) - pending_job_count().
 
 -spec running_jobs() -> [#job{}].
 running_jobs() ->
-    MatchSpec = [{#job{pid = '$1', _='_'},
-        [{is_pid,'$1'}],
-        ['$_']}
-    ],
-    ets:select(?MODULE, MatchSpec).
+    ets:tab2list(?MODULE) -- pending_jobs().
 
 
 -spec pending_job_count() -> non_neg_integer().
 pending_job_count() ->
-    Now = unix_now(),
-    % Select jobs where the pid is undefined and the exponential backoff
-    % threshold has been crossed.
-    MatchSpec = [{#job{pid = '$1',errored_at = '$2',errors = '$3', _='_'},
-        [{'not',{is_pid,'$1'}},
-        {'<',{'-',{'*',{'bsl',1,'$3'},?ERROR_BACKOFF_MILLIS},
-        {'*',{'-',{const,Now},'$2'},1000}},
-        0}],
-        [true]}
-    ],
+    MatchSpec = [{#job{pid='$1', _='_'}, [{'not', {'is_pid', '$1'}}], [true]}],
```

Comment:
Should this  be `#job{pid=undefined, ...}` ? 

Going by idea that pending jobs are the ones which are not running but a ready to run, so they don't have a pid yet.